python/cuml/test/test_pickle.py

```diff
@@ -131,49 +131,6 @@ def make_dataset(datatype, nrows, ncols, n_info):
     return X_train, y_train, X_test
 
 
-@pytest.mark.parametrize('datatype', [np.float32])
```

Comment:
I believe this test should still be in there

Reply:
Yes! I'll add it back